storybook/StoryBook.pro

```diff
@@ -0,0 +1,72 @@
+QT += core \
+    quick \
+    quickcontrols2 \
+    svg \ #
```

Comment:
I wonder how much we pay for svg if it requires those three modules in terms of app size. GUI is probably quite heavy stuff.

Reply:
GUI is always needed for QML apps; xml needed for svg

Reply:
I had an impression that gui is not needed, but you are right from what ldd shows :)